std_msgs/Header header
gvgexploration/RobotSignal[] signals